app/src/main/java/com/group2/reciperepositoryapp/DatabaseHelper.java

```diff
@@ -70,4 +70,15 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
 
     //Guys use this class to put your CRUD methods for your use
     //Start here and below
+
+    public void addIngrRecQuantity() {
+        String sql4 = "INSERT INTO CATEGORY_TABLE(Name) VALUES(?)";//QUERY TO INSERT CATEGORIES TO CATEGORY TABLE
+        String getCATID="SELECT CATEGORY_ID FROM CATEGORY_TABLE WHERE CATEGORY_ID =(SELECT COUNT(CATEGORY_ID) FROM CATEGORY_TABLE)";
+        String sql1 = "INSERT INTO INGREDIENT_TABLE(Name) VALUES(?)";//QUERY TO INSERT INGREDIENTS TO INGREDIENT TABLE
+        String getiNGREDIENTID="SELECT INGREDIENT_ID FROM INGREDIENT_TABLE WHERE INGREDIENT_ID =(SELECT COUNT(INGREDIENT_ID) FROM RECIPE_TABLE)";
+        String sql2 = "INSERT INTO RECIPE_TABLE(Name, Description,category_id,user_id) VALUES(?,?,?,?)";//QUERY TO INSERT RECIPES TO RECIPES TABLE
+        String getRecipeID="SELECT RECIPE_ID FROM RECIPE_TABLE WHERE RECIPE_ID =(SELECT COUNT(RECIPE_ID) FROM INGREDIENT_TABLE)";
+
+        String sql3 = "INSERT INTO RECIPE_INGREDIENT_TABLE (Recipe_id,Ingredient_id,Quantity) VALUES(?,?,?);";
+    }
```

Comment:
In creating CRUD methods in ANDROID JAVA and SQLite context, you need to follow the following steps:
1. Create a query string, e.g `String sql2 = "INSERT INTO RECIPE_TABLE(Name, Description,category_id,user_id) VALUES(?,?,?,?)";`
2. Create an object of an SQLiteDatabase, e.g `SQLiteDatabase db = this.getWritableDatabase();` for INSERT, DELETE and UPDATE QUERIES. For SELECT QUERIES; `SQLiteDatabase db = this.getReadabableDatabase();`
3. For INSERT, DELETE and UPDATE QUERIES you create an object from ContentValues class e.g `ContentValues values = new ContentValues();` and for SELECT QUERIES, you create an object from Cursor class e.g `Cursor cursor = db.rawQuery(Argument1 ,Argument2);` where Argument1 is the SQLQuery and Argument2 is an array of passed values in that SQLQuery.
4. For INSERT, DELETE and UPDATE QUERIES use the created ContentValues object to add the values (refer to the example below). For SELECT QUERIES you use the Cursor object to loop around the values (refer to the example below).

Examples: 
`//    Insert User
    public boolean addUser(String reg_no, String fname, String mname, String lname, String gen,
                           String b_date, String email, String phone, String region,
                           String district, String ward, String username, String role, String pass) {
        SQLiteDatabase db = this.getWritableDatabase();
        ContentValues values = new ContentValues();
        ContentValues value2 = new ContentValues();

        values.put(REG_Number, reg_no);
        values.put(FirstName, fname);
        values.put(MiddleName, mname);
        values.put(LastName, lname);
        values.put(Gender, gen);
        values.put(Bdate, b_date);
        values.put(Email, email);
        values.put(PhoneNumber, phone);
        values.put(Region, region);
        values.put(District, district);
        values.put(Ward, ward);


        long result = db.insert(REG_TABLE, null, values);

        value2.put(Username, username);
        value2.put(Role, role);
        value2.put(Password, pass);

        long result2 = db.insert(LOGIN_TABLE, null, value2);

        db.close();

        if (result != -1 && result2 != -1) return true;
        else return false;
    }`

`public ArrayList<String> getStaffs(){
        ArrayList<String> list = new ArrayList<String>();
        SQLiteDatabase db = this.getReadableDatabase();

        String staffQuery = "SELECT FirstName FROM user_reg,login_cred WHERE role='Staff' AND login_cred.Username = user_reg.REG_Number";

        Cursor cursor = db.rawQuery(staffQuery,null);

        if (cursor.getCount() > 0){
            while(cursor.moveToNext()){
                String staffs = cursor.getString(cursor.getColumnIndex("FirstName"));
                list.add(staffs);
            }
        }
        cursor.close();
        db.close();

        return list;
    }
`